apps/web/src/components/nav/NavMenuButton/NavMenuLinkButton.tsx

```diff
@@ -0,0 +1,55 @@
+import { FC, PropsWithChildren, useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { css } from '../../../styled-system/css';
+import { HStack } from '../../../styled-system/jsx';
+import { INavMenuButtonProps, rawButtonBaseStyles, ButtonLabel } from './NavMenuButton.shared';
+
+const rawLinkButtonStyles = css.raw({
+  '& _active, &.active': {
+    _before: {
+      content: '""',
+      position: 'absolute',
+      width: '50',
+      top: 0,
+      right: 0,
+      bottom: 0,
+      left: 0,
+      borderRadius: '7px 0px 0px 7px',
+      bgGradient: 'vertical',
+    },
+  },
+});
+
+interface INavMenuLinkButtonProps extends INavMenuButtonProps {
+  link: string;
+}
+
+export const NavMenuLinkButton: FC<PropsWithChildren<INavMenuLinkButtonProps>> = ({
+  rightSide,
+  testId,
+  icon,
+  link,
+  label,
+  isVisible = true,
+}) => {
+  const [popoverOpened, setPopoverOpened] = useState(false);
+  const [isHovered, setIsHovered] = useState<boolean>(false);
```

Comment:
not used?

Reply:
They're used in the next phase since we need the one-off visibility icon